oep-template.rst

```diff
@@ -0,0 +1,85 @@
+=================================
+OEP-1: OEP Purpose and Guidelines
+=================================
+
++---------------+-------------------------------------------+
+| OEP           | <link to OEP>                             |
++---------------+-------------------------------------------+
+| Title         | <OEP title>                               |
++---------------+-------------------------------------------+
+| Version       | <version string>                          |
++---------------+-------------------------------------------+
+| Last-Modified | <date string>                             |
++---------------+-------------------------------------------+
+| Author        | <list of authors' real names and          |
+|               | optionally, email addrs>                  |
++---------------+-------------------------------------------+
+| Arbiter       | <OEP czar's real name>                    |
++---------------+-------------------------------------------+
+| Status        | <Draft | Accepted | Deferred |            |
+|               | Rejected | Withdrawn | Final |            |
+|               | Replaced>                                 |
++---------------+-------------------------------------------+
+| Type          | <Architecture | Best Practice |           |
+|               | Process>                                  |
++---------------+-------------------------------------------+
+|  Created      | <date created on, in dd-mmm-yyyy format>  |
++---------------+-------------------------------------------+
+| `Replaces`    | <OEP number>                              |
++---------------+-------------------------------------------+
+| `Replaced-By` | <OEP number>                              |
++---------------+-------------------------------------------+
+
+Abstract
+========
+a short (~200 word) description of the technical issue being addressed.
+
+Specification
+=============
+The technical specification should describe the syntax and semantics of any new API.
+
+Motivation
+==========
+The motivation is critical for OEPs that want to change Open edX. It should
+clearly explain why the existing achictecture or process is inadequate to
+address the problem that the OEP solves, or why Open edX would be significantly
+improved by adopting the best practice. OEP submissions without sufficient
+motivation may be rejected outright.
+
+Rationale
+=========
+The rationale fleshes out the specification by describing what motivated the
+design and why particular design decisions were made. It should describe
+alternate designs that were considered and related work, e.g. how the
+feature is supported in other systems.
+
+The rationale should provide evidence of consensus within the community
+and discuss important objections or concerns raised during discussion.
```

Comment:
Somewhere should be a link to the discussion that hammered out some of the details.